### PR TITLE
refactor SVG decorations to .svg files

### PR DESCRIPTION
**Pull request by hasparus**
per @dimaMachina's request, refactored the landing page decorations to SVG files

### CODE REVIEW

**Review by saihaj — APPROVED** [review]

File: packages/components/src/components/decorations/arch-decoration-gradient-defs.svg [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<svg width="432" height="432" viewBox="0 0 432 432" class="absolute -z-10">
```
Comment (dimaMachina) [inline packages/components/src/components/decorations/arch-decoration-gradient-defs.svg:1]: it was supposed to be just transparent for this svg?

File: packages/components/src/components/decorations/highlight-decoration.svg [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<svg width="895" height="674" viewBox="0 0 895 674" fill="#86B6C1" class="firefox-highlight-fix">
```
Comment (dimaMachina) [inline packages/components/src/components/decorations/highlight-decoration.svg:1]: was supposed to be just transparent for this svg?

Comment (dimaMachina) [reply packages/components/src/components/decorations/highlight-decoration.svg:1]: I see the change in storybook, nice! 
image

File: packages/components/src/components/decorations/arch-decoration-gradient-defs.svg (hunk 1 above)
Comment (hasparus) [reply packages/components/src/components/decorations/arch-decoration-gradient-defs.svg:1]: This one holds just definitions of gradients so we don't repeat them multiple times in the DOM.

**Pull request merged**